proposals/description/tip_121b.ts

```diff
@@ -0,0 +1,180 @@
+      arguments: (addresses) => [
+        addresses.aaveEthPCVDeposit, // pcvDeposit
+        addresses.ethLidoPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `Move all ETH from aaveEthPCVDeposit to ethLidoPCVDeposit to deposit()`
+    },
+    {
+      target: 'ethLidoPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: (addresses) => [],
+      description: `Deposit ETH in Lido`
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: `Pause aaveEthPCVDeposit`
+    },
+
+    // 3. Cleanup FEI/TRIBE on TC Timelock from the Rari Infra team clawback
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.core, TC_TRIBE_BALANCE],
+      description: 'Move all 2.7M TRIBE from TC Timelock to DAO Treasury'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.feiDAOTimelock, TC_FEI_BALANCE],
+      description: 'Move all 2.7M FEI from TC Timelock to DAO Timelock'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'burn(uint256)',
+      arguments: (addresses) => [TC_FEI_BALANCE],
+      description: 'Burn 2.7M FEI from the DAO Timelock'
+    },
+    // 4. Update CR oracle to use stETH oracle for stETH rather than ETH oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.weth, addresses.chainlinkStEthUsdOracleWrapper],
+      description: 'Update the ETH oracle in the CR oracle, to the stETH oracle as remaining ETH will be stETH'
+    },
+    // 5. Update CR oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: (addresses) => [[addresses.lusdHoldingPCVDeposit]],
+      description: 'Add smart contracts to CR Oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.aaveFeiPCVDepositWrapper,
+          addresses.compoundFeiPCVDepositWrapper,
+          addresses.lusdToDaiLensLusd,
+          addresses.lusdToDaiLensDai,
+          addresses.tribalCouncilTimelockFeiLens
+        ]
+      ],
+      description: 'Remove deprecated/empty smart contracts from CR Oracle'
+    },
+    // 6. Remove non-needed safe addresses
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [
+        [addresses.voltHoldingPCVDeposit, addresses.gOHMHoldingPCVDeposit, addresses.lusdHoldingPCVDeposit]
```

Comment:
If we're keeping the `lusdHoldingPCVDeposit` in the CR, probably should keep as a safe address also?

Reply:
done